### PR TITLE
Even Moar Donations! 

### PR DESCRIPTION
**Pull request by StakeWithPride**
Several new ones so I'm finally caught up.

### CODE REVIEW

File: PRIDE/donations.yml [hunk 1]
```diff
@@ -36,4 +36,134 @@
...
+  ada: 76
+  usd: 119.73
+  timestamp: 1621624087
+  files:
+    - PRIDE/prideMay21.png
+
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1399048939454623751
+  recipient: MBARARA RISE FOUNDATION
```
Comment (ResponsibleStaking) [inline PRIDE/donations.yml:108]: This is not valid against YAML conventions.
If you only want to use this recipient once you can just delete the name in this line.
If you want to reuse it you can give it a name like &mbaara and reuse it in later donations later with *mbaara
When reusing you will not need the sub-attributes (name, contact, ..)

File: PRIDE/donations.yml [hunk 2]
```diff
@@ -36,4 +36,134 @@
...
+    address: n/a
+  ada: 80
+  usd: 120 
+  timestamp: 1622488087
+  files:
+    - PRIDE/prideMay31.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1403399311300255746
+  recipient: LGBT+ Rights Ghana
```
Comment (ResponsibleStaking) [inline PRIDE/donations.yml:121]: Same Issue as above

File: PRIDE/donations.yml [hunk 3]
```diff
@@ -36,4 +36,134 @@
...
+    address: n/a
+  ada: 70
+  usd: 100
+  timestamp: 1623438487
+  files:
+    - PRIDE/prideJune11.png
 
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1405633561231724551
+  recipient: LGBT VOICE TANZANIA
```
Comment (ResponsibleStaking) [inline PRIDE/donations.yml:134]: same

File: PRIDE/donations.yml [hunk 4]
```diff
@@ -36,4 +36,134 @@
...
+    address: n/a
+  ada: 89
+  usd: 120
+  timestamp: 1622401687
+  files:
+    - PRIDE/prideMay30.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1409220479458545675
+  recipient: All Out
```
Comment (ResponsibleStaking) [inline PRIDE/donations.yml:147]: Same

File: PRIDE/donations.yml [hunk 5]
```diff
@@ -36,4 +36,134 @@
...
+    address: n/a
+  ada: 80
+  usd: 100
+  timestamp: 1624820887
+  files:
+    - PRIDE/prideJune27.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1413508489654063105
+  recipient: Iranti
```
Comment (ResponsibleStaking) [inline PRIDE/donations.yml:160]: Same

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]
Invalid YAML Structure. Please update

**Comment by StakeWithPride** [comment]
Thanks for the feedback. I've updated the yaml by removing the name attribute. Thanks for your guidance.

**Review by ResponsibleStaking — APPROVED** [review]

**Pull request merged**